ga4gh/backend.py

```diff
@@ -23,14 +24,9 @@ class AbstractBackend(object):
     This class provides methods for all of the GA4GH protocol end points.
     """
     def __init__(self):
-        self._variantSetIdMap = {}
-        self._variantSetIds = []
-        self._referenceSetIdMap = {}
-        self._referenceSetIds = []
-        self._readGroupSetIdMap = {}
-        self._readGroupSetIds = []
-        self._callSetIdMap = {}
-        self._callSetIds = []
+        self._variantSetIdMap = collections.OrderedDict()
+        self._referenceSetIdMap = collections.OrderedDict()
+        self._readGroupSetIdMap = collections.OrderedDict()
```

Comment:
`OrderedDict` will be very slow - below is a link to a quick explaination:

http://stackoverflow.com/questions/18422995/why-is-ordereddict-10x-slower-than-dict-and-list

~p